integrations/dbolya-yolact/README.md

```diff
@@ -40,30 +40,55 @@ python3 -m venv venv # create a venv virtual environment
 source venv/bin/activate # activate venv
 pip install --upgrade pip # upgrade pip
 ```
-To begin, run the following command in the root directory of this integration 
-(`cd integrations/dbolya-yolact`):
+To begin, install `sparseml>=1.1`
+
 ```bash
-bash setup_integration.sh
+pip install "sparseml>=1.1"
 ```
 
-The `setup_integration.sh` file will clone the yolact-sparseml integration repository. After the repo has successfully cloned, all dependencies from the `yolact/requirements.txt` file will install in your current environment.
-
 Note: This integration requires `python>=3.7,<3.10`
 
 
 ## Quick Tour
 
+### Downloading COCO
+
+The `sparseml.yolact.download` utility provides an easy to use interface to download
+the `COCO` dataset.
+
+Simply invoke:
+```bash
+sparseml.yolact.download
+```
+
+To download `COCO` test-set invoke `sparseml.yolact.download --test`; for more information
+on this utility invoke it with the `--help` option. (Kindly note the download location of the dataset)
```

Comment:
nit: using `invoke` twice.

Reply:
I get that, but the sentence still sounds good to me; any alternate suggestions on this @jeanniefinks ?